clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -7,12 +7,13 @@
   {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
 # Test Variales
-  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 1}}
```

Comment:
I actually wanted to ask about this: `DefaultParam` is supposed to make it possible to provide those values via environment variables instead of overrides, right?

I'm following the pattern from `NODES_PER_NAMESPACE` here, but when I tested it, I found that the env variables are ignored. Is that perhaps because they don't start with `CL2_`?

That's broken then also for the other variables, something that I should fix before repeating the same mistake.

Reply:
Yeah - only the ones prefixed with CL2_ are propagated down the stack (I can't remember the exact motivation for that, but there was a good reason to not propagate everything).

Reply:
I decided to stay with the current naming, i.e. env variables continue to not work...